Resources/public/js/controller/custom_entity-edit.js

```diff
@@ -20,19 +20,19 @@ define(
                         return createForm.call(
                             this,
                             this.$el,
-                            normalizedEntity.data,
+                            normalizedEntity,
                             normalizedEntity.meta.form
                         );
                     });
 
-                function createForm(domElement, entity, formExtension) {
+                function createForm(domElement, normalizedEntity, formExtension) {
```

Comment:
I needed to change this in order to get my project to save. Without it the javascript kept throwing an exception on the cache invalidator. I've been trying to understand what caused this new issue but I have no idea. 

https://github.com/akeneo/pim-community-dev/blob/bd275d539742951446f305ea179f8522fe6ae88d/src/Pim/Bundle/EnrichBundle/Resources/public/js/form/cache-invalidator.js#L30